website/frontend/src/components/blocks/Header/children/Menu/Menu.tsx

```diff
@@ -14,20 +15,26 @@ const Menu = ({ openSidebar }: Props) => {
 
   return (
     <ul className={styles.menu}>
-      <li>
+      <li className={styles.item}>
         <span className={styles.link} onClick={openSidebar}>
           {isApiReady ? localStorage.chain : 'Loading...'}
         </span>
       </li>
-      <li>
-        <Link to="/explorer" className={styles.link}>
-          Explorer
-        </Link>
+      <li className={styles.item}>
+        <NavLink
+          className={({ isActive }) => (isActive ? clsx(styles.link, styles.active) : styles.link)}
+          to={routes.explorer}
+        >
+          <span>Explorer</span>
+        </NavLink>
       </li>
```

Comment:
```suggestion
      const linkClassName = ({ isActive }: ClassNameProps) => clsx(styles.link, isActive && styles.active);
      
      <li>
        <NavLink to={routes.explorer} className={linkClassName}>
          Explorer
        </NavLink>
      </li>
```
- Getting className should be done via function since it's the same across all links
- No need to style `li`
- No need for inner `span`